pkg/ibmvpc/vpc.go

```diff
@@ -493,6 +513,10 @@ func (sgl *SecurityGroupLayer) Name() string {
 	return ""
 }
 
+func (sgl *SecurityGroupLayer) ExtendedName(config *vpcmodel.VPCConfig) string {
+	return ""
+}
+
```

Comment:
```suggestion
```
same here

Reply:
also here, I think this is the best option. Lets discuss in the last comment